examples/jss/src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { render } from 'react-dom';
 
-import './global-styles';
+import GlobalStyles from './components/global-styles';
 import App from './components/app';
 
-render(<App />, document.getElementById('example-root'));
+render(<GlobalStyles><App /></GlobalStyles>, document.getElementById('example-root'));
```

Comment:
For example, something like this would be really declarative and make sense in the JSX ≈ DOM model:

```JSX
import { Style } from 'jss' ;

<Style>
{/* Global styles here *} 
</Style>
```